src/styles/shared/_index.scss

```diff
@@ -0,0 +1,2 @@
+@use './components/Buttons' as buttons;
+@include buttons.styles;
```

Comment:
It wasn't immediately obvious to me what this line was doing. Per my comments on the JS, I suggest that we restrict `./components/CamelCase` naming to modules that define a single component.